### ISSUE TITLE
Update links that are pointing to the Diagram Center to PCG area

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#99]
There are links in the document that point to the Diagram Center. The link does indicate that the repo has been moved to the W3C, but these links should be updated in the document.

**Comment by clapierre** [comment]
Complete via Merge above.
